game/templates/game/stats.html

```
@@ -53,7 +56,20 @@ <h2>Recent Games</h2>
                     <td>{{ result.get_mode_display }}</td>
                     <td>{{ result.get_winner_display }}</td>
                     <td>{{ result.get_end_reason_display }}</td>
+                    <td>{{ result.moves|length }}</td>
                     <td>{{ result.played_at|date:"Y-m-d H:i" }}</td>
+                    <td>
+                        <button
+                            class="details-btn"
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)